scripts/build

```diff
@@ -40,7 +45,19 @@ if [ -r "$REQ_PACKAGES" ] && [ "$REQ_PACKAGES" -nt "chroot-$DIST/home/user/.inst
     sed "s/DIST/$DIST/g" "$REQ_PACKAGES" > build-pkgs-temp.list
     echo "-> Installing $COMPONENT build dependencies in $DIST environment"
     TMP_PACKAGES_ARR=( $(cat build-pkgs-temp.list) )
-    sudo chroot "$PWD/chroot-$DIST" yum install -y "${TMP_PACKAGES_ARR[@]}"
+    if [ "x${FEDORA_MIRROR}" != "x" ]; then
+        sudo awk -i inplace '
+        BEGIN {
+            mirror=ARGV[1];     delete ARGV[1];
+            releasever=ARGV[2]; delete ARGV[2];
+        }
+        {
+            gsub("^baseurl=.*/(linux|fedora)/", "baseurl=" mirror "/");
+            gsub("\\$releasever", releasever);
+            print;
+        }' "${FEDORA_MIRROR%/}" "${DIST#fc}" "$PWD/chroot-$DIST/etc/yum.repos.d/fedora.repo" "$PWD/chroot-$DIST/etc/yum.repos.d/fedora-updates.repo"
```

Comment:
Is it really necessary to duplicate it here? `prepare-chroot-base` in builder-rpm (which definitely was called, as chroot wouldn't exists otherwise) already setup mirror if FEDORA_MIRROR is set.

Reply:
If I understand correctly, it setup mirror for the bootstrapping. Then, in the prepare_chroot_builder it removes only the use of metalinks. Then, if FEDORA_MIRROR is set, it passes the url in the dnf options each time. So no persistent configuration. So maybe we should directly modify the chroot env in the prepare_chroot_builder?

Reply:
By the way, the default baseurl for updats has changed. "Everything" is also added now at the end before the basearch.